video_xblock/static/js/studio-edit.js

```diff
@@ -357,6 +359,67 @@ function StudioEditableXBlock(runtime, element) {
     });
 
     /**
+     * Get transcripts from 3playmedia's API and show result message.
+     */
+    function getTranscripts3playmediaApi(data) {
```

Comment:
This function should be simplified by moving `showStatus` call into `.always()` part of `$.ajax` deferred.
See [here](https://github.com/raccoongang/xblock-video/blob/dev/video_xblock/static/js/studio-edit.js#L407-L446) for the reference.